test/testdrive-old-kafka-src-syntax/source-tables.td

```diff
@@ -47,9 +47,16 @@ ALTER SYSTEM SET enable_load_generator_key_value = true
 > SELECT count(*) FROM bids
 255
 
+# TODO(def-) Reenable when database-issues#8703 is fixed
+$ skip-if
+SELECT true
+
 > SELECT status FROM mz_internal.mz_source_statuses WHERE name = 'bids';
 running
 
+# TODO(def-) Reenable when database-issues#8703 is fixed
+$ skip-end
+
```

Comment:
<s>I don't think this should be necessary since this is not using the `CREATE TABLE FROM SOURCE` syntax?</s> Oh, ok, this single file does. We could also delete this file.